core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/logprovider/recoverer.go

```diff
@@ -687,6 +691,7 @@ func (r *logRecoverer) removePending(workID string) {
 		}
```

Comment:
Use `prommetrics.AutomationRecovererPendingPayloads.Dec()` instead of  `prommetrics.AutomationRecovererPendingPayloads.Set(float64(len(r.pending)))`